src/lib.rs

```diff
@@ -1231,6 +1231,10 @@ impl CofactorGroup for ExtendedPoint {
     fn into_subgroup(self) -> CtOption<Self::Subgroup> {
         CtOption::new(SubgroupPoint(self), self.is_torsion_free())
     }
+
+    fn is_torsion_free(&self) -> Choice {
+        self.is_torsion_free()
+    }
```

Comment:
Are we sure this isn't recursing forever and nothing is calling it in our test suite? Might be better to do `ExtendedPoint::is_torsion_free(self)`.

Reply:
It's a call to a concrete method inside a trait method, which rustc does know how to handle. The stack overflow issue we saw in the Zcash crates was due to two traits having the same method name.

Reply:
From https://doc.rust-lang.org/book/ch19-03-advanced-traits.html#fully-qualified-syntax-for-disambiguation-calling-methods-with-the-same-name:

> When we call `fly` on an instance of `Human`, the compiler defaults to calling the method that is directly implemented on the type
> [...]
> Specifying the trait name before the method name clarifies to Rust which implementation of `fly` we want to call. We could also write `Human::fly(&person)`, which is equivalent to the `person.fly()` that we used in Listing 19-18, but this is a bit longer to write if we don’t need to disambiguate.

Reply:
Okay, cool.